build.sh

```diff
@@ -801,6 +801,39 @@ case "$COMMAND" in
         ;;
 
     ######################################
+    # Continuous Integration
+    ######################################
+
+    "ci-pr")
+        mkdir -p build/reports
+
+        if [ "$target" = "docs" ]; then
+            ./build.sh set-swift-version
+            ./build.sh verify-docs
+        elif [ "$target" = "swiftlint" ]; then
+            ./build.sh verify-swiftlint
+        else
+            sha=$ghprbSourceBranch
```

Comment:
This doesn't appear to be used, and shouldn't be needed (since we need to have already downloaded the correct version to be running this...)